.circleci/images/x86_64-linux/Dockerfile

```diff
@@ -1,10 +1,23 @@
-FROM haskell:8.2
+FROM debian:jessie
```

Comment:
Btw, if you use debian:stretch, you can make use of http://downloads.haskell.org/debian/